Projects/Home/Sources/Models/Questions.swift

```diff
@@ -1,484 +1,638 @@
-            questionEn: L10n.hcClaimsQ12_en,
-            answer: L10n.hcClaimsA12,
-            topicType: .claims
-        )
+extension Sequence where Iterator.Element: QuestionProtocol {
+    func asQuestions() -> [Question] {
+        return map({ $0.asQuestion() })
     }
 }
 
-enum CoverageQuestions {
-    static var q1: Question {
-        return .init(
-            question: L10n.hcCoverageQ01,
-            questionEn: L10n.hcCoverageQ01_en,
-            answer: L10n.hcCoverageA01,
-            topicType: .coverage
-        )
-    }
-    static var q2: Question {
-        return .init(
-            question: L10n.hcCoverageQ02,
-            questionEn: L10n.hcCoverageQ02_en,
-            answer: L10n.hcCoverageA02,
-            topicType: .coverage
-        )
-    }
-    static var q3: Question {
-        return .init(
-            question: L10n.hcCoverageQ03,
-            questionEn: L10n.hcCoverageQ03_en,
-            answer: L10n.hcCoverageA03,
-            topicType: .coverage
-        )
-    }
-    static var q4: Question {
-        return .init(
-            question: L10n.hcCoverageQ04,
-            questionEn: L10n.hcCoverageQ04_en,
-            answer: L10n.hcCoverageA04,
-            topicType: .coverage
-        )
-    }
-    static var q5: Question {
-        return .init(
-            question: L10n.hcCoverageQ05,
-            questionEn: L10n.hcCoverageQ05_en,
-            answer: L10n.hcCoverageA05,
-            topicType: .coverage
-        )
-    }
-    static var q6: Question {
-        return .init(
-            question: L10n.hcCoverageQ06,
-            questionEn: L10n.hcCoverageQ06_en,
-            answer: L10n.hcCoverageA06,
-            topicType: .coverage
-        )
-    }
-    static var q7: Question {
-        return .init(
-            question: L10n.hcCoverageQ07,
-            questionEn: L10n.hcCoverageQ07_en,
-            answer: L10n.hcCoverageA07,
-            topicType: .coverage
-        )
-    }
-    static var q8: Question {
-        return .init(
-            question: L10n.hcCoverageQ08,
-            questionEn: L10n.hcCoverageQ08_en,
-            answer: L10n.hcCoverageA08,
-            topicType: .coverage
-        )
-    }
-    static var q9: Question {
-        return .init(
-            question: L10n.hcCoverageQ09,
-            questionEn: L10n.hcCoverageQ09_en,
-            answer: L10n.hcCoverageA09,
-            topicType: .coverage
-        )
-    }
-    static var q10: Question {
-        return .init(
-            question: L10n.hcCoverageQ10,
-            questionEn: L10n.hcCoverageQ10_en,
-            answer: L10n.hcCoverageA10,
-            topicType: .coverage
-        )
-    }
-    static var q11: Question {
-        return .init(
-            question: L10n.hcCoverageQ11,
-            questionEn: L10n.hcCoverageQ11_en,
-            answer: L10n.hcCoverageA11,
-            topicType: .coverage
-        )
+enum PaymentsQuestions: CaseIterable {
+    case q1
```

Comment:
could we rename these to be something like case paymentsQuestion1, paymentsQuestion2 etc? Just to not be confused